apps/health/boot.js

```diff
@@ -92,3 +96,22 @@ Bangle.on("health", health => {
     health.movement /= health.movCnt;
   require("Storage").write(fn, getRecordData(health), sumPos, DB_FILE_LEN);
 });
+
+function handleStepGoalNotification() {
+  var settings = require("Storage").readJSON("health.json",1)||{};
+  const steps = Bangle.getHealthStatus("day").steps;
+  if (settings.stepGoalNotification && settings.stepGoal > 0 && steps >= settings.stepGoal) {
+    const now = new Date(Date.now()).toISOString().split('T')[0]; // yyyy-mm-dd
+    if (!settings.stepGoalNotificationDate || settings.stepGoalNotificationDate < now) { // notification not yet shown today?
+      Bangle.buzz(200, 0.5);
+      /*LANG*/E.showPrompt("You reached your daily goal of " + settings.stepGoal + " steps!", {
+        /*LANG*/title: "Step goal reached!",
```

Comment:
Similarly I think the /\*LANG\*/ tag should be just before the string and not before title, or will this work?